src/features/locators/types/locator.types.ts

```diff
@@ -90,7 +88,7 @@ export interface Locator extends PredictedEntity {
   active?: boolean;
   isCustomName?: boolean;
   isCustomLocator?: boolean;
-  locatorType?: LocatorType;
+  locatorType: LocatorType;
```

Comment:
it was optional for not to store useless data. Why you made it required?

Reply:
I am not sure that locatorType is useless data.
When user generates or edits PO, creates or edits locator this field can't be empty, so mark it as optional looks unlogical for me.
I offer to make it required at `PageObject` interface at list, if you don't want always to store `locatorType` in `Locator` interface. What do you think?